classes/Hook/HookDisplayFooter.php

```diff
@@ -109,7 +159,7 @@ public function run()
             if ($this->module->eligible == 0) {
                 $gaScripts .= $gaTools->addProductImpression($products);
             }
-            $gaScripts .= $gaTools->addProductClick($products);
+            $gaScripts .= $gaTools->addProductClick($products, $currencyIsoCode);
```

Comment:
@Progi1984 `$this->context->currency->iso_code`

Reply:
Fixed